CDCgov/NEDSS-Modernization
# CNFT2-1223 Manage Pages routing

## Description

When clicking the `Manage Pages` link in the Classic™ System management screen (see below), the user should be sent to the modernized Page Library screen.

![image](https://github.com/CDCgov/NEDSS-Modernization/assets/109251240/0fc3f59d-096c-4b3a-947c-be438265fca1)


## Tickets

* [CNFT2-1223](https://cdc-nbs.atlassian.net/browse/CNFT2-1223)

## Steps to verify

1. Rebuild modernization-api and nbs-gateway projects
2. Navigate to http://localhost:8080/nbs/login
3. Login as `superuser`
4. Click `System Management`
5. Click `Manage Pages`
6. Verify Modernized Page Library screen is loaded

![manage-page-redirect](https://github.com/CDCgov/NEDSS-Modernization/assets/109251240/b3a4ab0d-10fa-4d90-9838-eda0ff6dac1d)


[CNFT2-1223]: https://cdc-nbs.atlassian.net/browse/CNFT2-1223?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
